examples/platform/telink/zephyr_ext/zephyr_key_matrix.c

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright (c) 2024 Telink Semiconductor
```

Comment:
I'm not sure whether this is OK to add files without `Copyright (c) 2024 Project CHIP Authors` copyright notice to the matter SDK project...

Reply:
Is this file copied from somewhere else, or did you author this from scratch? If this is copied, I think you need to retain the old copyright and add the Project CHIP Authors copyright.

Although someone more familiar should confirm

Reply:
@arkq @tehampson 
All new files in this PR were written by Telink employee @andriy-bilynskyy

Reply:
Apache 2.0 license should not have the "All rights reserved." phrase. I can see that some files in the project have that phrase, but I think that's actually an error (such license is not apache 2.0 anymore).

Reply:
@arkq Fixed